Sources/BraintreeCoreSwift/BTClientTokenError.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+
+///  Error codes associated with a client token.
+@objc public enum BTClientTokenError: Int, Error, CustomNSError, LocalizedError {
+
+    /// Authorization fingerprint was not present or invalid
+    case invalidAuthorizationFingerprint
+
+    /// Config URL was missing or invalid
+    case invalidConfigURL
+
+    /// Invalid JSON
+    case invalidJSON
+
+    /// Invalid client token format
+    case invalidFormat
+
+    /// Unsupported client token version
+    case unsupportedVersion
+    
+    /// Invalid UTF8 Encoding
+    case expectedUTF8Encoding
```

Comment:
Super nitpick - but these would make more sense to me phrased like `failedUTF8Decoding`, since the decoding is what failed. We don't necessarily know the encoding is wrong.